client/blocks/login/index.jsx

```diff
@@ -97,8 +97,8 @@ class Login extends Component {
 		isPartnerSignup: PropTypes.bool,
 		loginEmailAddress: PropTypes.string,
 		action: PropTypes.string,
-		isGravatar: PropTypes.bool,
-		isGravatarLoginPage: PropTypes.bool,
+		isGravPoweredClient: PropTypes.bool,
```

Comment:
Minor thing: I would prefer to use the full name for Gravatar instead of "Grav". It makes sense now, but I can see myself wondering what is "Grav" in a couple of years hehe

Feel free to ignore if "Grav" is a famous shortname for Gravatar (like Woo for WooCommerce). I am not that familiar with Gravatar (yet!).

Reply:
"Grav" is the shorthand of "Gravatar", I use it to make a difference with the  `isGravatar ` flag.

Reply:
I agree with @aaronfc: I'd much prefer if we keep the full `gravatar` in places. I don't think `grav` is a commonly used shorthand in the codebase.